### PR TITLE
mgr/orch: remove ansible and deepsea

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: src/pybind/mgr/orchestrator_cli/module.py [hunk 1]
```diff
@@ -333,6 +333,15 @@ def ukn(s):
...
 
+    @orchestrator._cli_write_command(
+        'orchestrator rm ',
+        "name=group,type=CephString",
+        'Remove a service group')
+    @orchestrator._cli_write_command(
+        'orchestrator rmi ',
+        "name=name,type=CephString",
+        'Remove a service instance')
+
```
Comment (sebastian-philipp) [inline src/pybind/mgr/orchestrator_cli/module.py]: remove this.

File: doc/mgr/orchestrator_modules.rst [hunk 2]
```diff
@@ -14,7 +14,7 @@ ceph-mgr orchestrator modules
 
 In this context, *orchestrator* refers to some external service that
 provides the ability to discover devices and create Ceph services.  This
-includes external projects such as ceph-ansible, DeepSea, and Rook.
+includes external projects such as DeepSea and Rook.
```
Comment (concubidated) [inline doc/mgr/orchestrator_modules.rst]: Minus DeepSea

Comment (liewegas) [reply doc/mgr/orchestrator_modules.rst]: fixed

**Comment by liewegas** [comment]
jenkins render docs

File: src/mon/OSDMonitor.cc [hunk 3]
```diff
@@ -5524,7 +5524,7 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
     }
     for (auto& k : {
 	"pod_name", "pod_namespace", // set by rook
-	"container_name"             // set by ceph-ansible
+	"container_name"             // set by cephadm, ceph-ansible
```
[the remark this answers is not in the document]
Comment (liewegas) [reply src/mon/OSDMonitor.cc]: ah, that's a bug.  Will fix!

**Pull request merged**